ui/overview.js

```diff
@@ -120,7 +120,7 @@ function _showWindows(overview) {
     _showOrSwitchPage(overview, ViewSelector.ViewPage.WINDOWS);
 }
 
-function toggleApps(overview) {
+function toggleApps(overview, ignoreNoWindowsDialog) {
```

Comment:
Probably better to pass a default value here: `ignoreNoWindowsDialog = false`

Reply:
Ok, will do - though if we don't pass anything the variable will be "undefined" which serves the same purpose.

Reply:
Done.

Reply:
Sorry, there were actually 2 places that needed updating, now it is really done.